src/fides/api/ops/api/v1/endpoints/privacy_experience_config_endpoints.py

```diff
@@ -0,0 +1,327 @@
+    *,
+    db: Session = Depends(deps.get_db),
+    experience_config_id: str,
+) -> PrivacyExperienceConfig:
+    """
+    Returns a PrivacyExperienceConfig.
+    """
+    logger.info("Retrieving experience config with id '{}'.", experience_config_id)
+    return get_experience_config_or_error(db, experience_config_id)
+
+
+@router.patch(
+    urls.EXPERIENCE_CONFIG_DETAIL,
+    status_code=HTTP_200_OK,
+    response_model=ExperienceConfigCreateOrUpdateResponse,
+    dependencies=[
+        Security(
+            verify_oauth_client,
+            scopes=[
+                PRIVACY_EXPERIENCE_UPDATE,
+            ],
+        )
+    ],
+)
+def experience_config_update(
+    *,
+    db: Session = Depends(deps.get_db),
+    experience_config_id: str,
+    experience_config_data: ExperienceConfigUpdate,
+) -> ExperienceConfigCreateOrUpdateResponse:
+    """
+    Update Experience Config and then attempt to upsert Experiences and link back to ExperienceConfig.
+
+    All regions that should be linked to this ExperienceConfig (or remain linked) need to be
+    included in this request.
+    """
+    experience_config: PrivacyExperienceConfig = get_experience_config_or_error(
+        db, experience_config_id
+    )
+    experience_config_data_dict: Dict = experience_config_data.dict(exclude_unset=True)
+    # Pop the regions off the request
+    regions: List[PrivacyNoticeRegion] = experience_config_data_dict.pop("regions")
+
+    # Because we're allowing patch updates here, first do a dry update and make sure the experience
+    # config wouldn't be put in a bad state.
+    dry_update: PrivacyExperienceConfig = experience_config.dry_update(
+        data=experience_config_data_dict
+    )
+    try:
+        ExperienceConfigCreate.from_orm(dry_update)
+    except ValueError as exc:
+        raise HTTPException(
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+            # pylint: disable=no-member
+            detail=exc.errors(),  # type: ignore
+        )
+
+    # Fail early if the ExperienceConfig is going to be incompatible with any of the regions
+    validate_experience_config_and_region_compatibility(
+        db=db,
+        component_type=dry_update.component,  # type: ignore[arg-type]
+        delivery_mechanism=dry_update.delivery_mechanism,  # type: ignore[arg-type]
+        regions=regions,
+    )
+
+    logger.info("Updating experience config of id '{}'.", experience_config.id)
+    experience_config.update(db=db, data=experience_config_data_dict)
+    db.refresh(experience_config)
+
+    current_regions: List[PrivacyNoticeRegion] = experience_config.regions
+    not_included_in_request: List[
+        PrivacyExperience
+    ] = remove_config_from_matched_experiences(
+        db,
+        experience_config,
+        [
+            PrivacyNoticeRegion(reg)
+            for reg in {reg.value for reg in current_regions}.difference(
+                {reg.value for reg in regions}
+            )
+        ],
+    )
```

Comment:
Here I remove any regions from the ExperienceConfig that weren't included in the request.  This just means I remove the FK between the this ExperienceConfig and any linked PrivacyExperiences: I never delete a PrivacyExperience.

Reply:
sure can!